tests/testthat/test-linear_pool.R

```diff
@@ -329,23 +327,23 @@ test_that("(weighted) quantiles correctly calculated - lognormal family", {
                                           model_id = "hub-ensemble",
                                           task_id_cols = NULL,
                                           n_samples = 1e5) |>
-                            dplyr::mutate(output_type_id = as.numeric(output_type_id))
+                            dplyr::mutate(output_type_id = (output_type_id))
```

Comment:
Is this line doing anything, or could we just delete it?

Reply:
Was trying to get rid of the coercion to numeric type for the `output_type_id` here and in other instances but it looks like I didn't properly delete the entire mutate statement. I've fixed this in a new commit